### PR TITLE
Feature/discovar

### PR DESCRIPTION
**Pull request by jerowe**
* [ ] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/discovar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+about:
+  home: https://www.broadinstitute.org/software/discovar
+  license: ''
+  summary: 'Suitable for variant calling with reference and de novo assembly of small genomes.'
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/discovar/meta.yaml]: build numbers starts with 0

File: recipes/discovar/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+about:
+  home: https://www.broadinstitute.org/software/discovar
+  license: ''
```
Comment (bgruening) [inline recipes/discovar/meta.yaml]: license is missing.

File: recipes/discovar/build.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -x -e
+
+export CC=${PREFIX}/bin/gcc
```
Comment (bgruening) [inline recipes/discovar/build.sh]: Wondering if this is really needed.

Comment (jerowe) [inline recipes/discovar/build.sh]: I think so. I want to make sure it uses the anaconda gcc

File: recipes/discovar/meta.yaml (hunk 2 above)
Comment (jerowe) [inline recipes/discovar/meta.yaml]: fixed and fixed

File: recipes/discovar/build.sh (hunk 3 above)
Comment (bgruening) [inline recipes/discovar/build.sh]: This should not be necessary, afaik.

Comment (jerowe) [inline recipes/discovar/build.sh]: I'll commit a build without it. I suppose you're right. Is ${PREFIX}/bin already the first entry in PATH?

**Comment by jerowe** [comment]
Merge?

**Comment by bgruening** [comment]
Thanks!

**Comment by jerowe** [comment]
Thanks!

**Pull request merged**